credsweeper/file_handler/file_path_extractor.py

```diff
@@ -105,3 +106,12 @@ def check_exclude_file(cls, config: Config, path: str) -> bool:
         if Util.get_extension(path) in config.exclude_extensions:
             return True
         return False
+
+    @classmethod
+    def check_file_size(cls, config: Config, path: str) -> bool:
+        if config.size_limit is None:
+            return False
+        if os.path.getsize(path) > config.size_limit * 1024 * 1024:  # Convert size_limit to byte
```

Comment:
may be use human readable arguments 1M 2Gb and so on?

Reply:
I added `humanfriendly` module for file size with [bf433d1](https://github.com/Samsung/CredSweeper/pull/125/commits/bf433d1977b7ecb73222f1e52aa99f864c3e8312).

Thank you for review.